internal/pkg/testhelpers/iptbtester/genesis.go

```diff
@@ -5,30 +5,49 @@ import (
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
+	"os"
 	"strconv"
 	"testing"
 	"time"
 
-	"github.com/stretchr/testify/require"
-
 	commands "github.com/filecoin-project/go-filecoin/cmd/go-filecoin"
 	"github.com/filecoin-project/go-filecoin/internal/pkg/constants"
 	"github.com/filecoin-project/go-filecoin/internal/pkg/types"
 	gengen "github.com/filecoin-project/go-filecoin/tools/gengen/util"
+	"github.com/stretchr/testify/require"
 )
 
 // GenesisInfo chains require information to start a single node with funds
```

Comment:
I'm guessing you checked if the testdaemon and fast don't give you this explicitly (esp since I think fast uses this) but wanted to double check because the necessity of doing work on this integration test framework directly means there are 4 places to worry about which I would love to avoid in the future.